<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>t3</class>
 <widget class="QWidget" name="t3">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>285</width>
    <height>296</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButtonDoIt">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>90</y>
     <width>141</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>pushButtonReplace</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonAdd">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>10</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>radioButtonAdd</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup</string>
   </attribute>
  </widget>
  <widget class="QRadioButton" name="radioButtonRemove">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>30</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>radioButtonRemove</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup</string>
   </attribute>
  </widget>
  <widget class="QTextEdit" name="outputTextEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>261</width>
     <height>161</height>
    </rect>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonReplace">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>50</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>RadioButton</string>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup</string>
   </attribute>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup">
   <property name="exclusive">
    <bool>true</bool>
   </property>
  </buttongroup>
 </buttongroups>
</ui>
